Dockerfile

```diff
@@ -21,12 +21,10 @@ RUN npm ci --ignore-scripts
 COPY --from=esy /app/_release /grain/node_modules/esy/platform-linux-arm64
 # Manually run esy's postinstall script
 RUN cd node_modules/esy && npm run postinstall
-
 # Necessary because we disabled scripts during the original install
-RUN npm run prepare
-
-# Build the compiler and CLI
-RUN npm run compiler build
+RUN npm run prepare \
+    # Build the compiler and CLI
+    && npm run compiler build
```

Comment:
We don't want to combine these because we want the results of the prepare script (which never changes) to be cached separately from the build (which changes on every commit).